src/CSET/recipes/extract_instant_model_level_air_temp.yaml

```
@@ -0,0 +1,36 @@
+title: Extract Meaned Model level Air Temperature
```

```suggestion
title: Meaned Model Level Air Temperature Spatial Plot
```
More accurate title.